.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./
         with:
-          sdk: dev
+          sdk: 'dev'
```

Comment:
Could be reverted now that we don't use quotes in the other files.

Reply:
👍